pkg/models/trb_request.go

```diff
@@ -135,13 +135,13 @@ type TRBRequestsByCedarSystemIDsRequest struct {
 
 type TRBRequestsByCedarSystemIDsResponse struct {
```

Comment:
Same here -- if you wouldn't mind just throwing a high-level comment on this type explaining what it's for, that would be appreciated!

Reply:
Added in [2f9554c](https://github.com/CMS-Enterprise/easi-app/pull/2727/commits/2f9554c1a63d663fae83cbb06de56c6c18c8b4ca)